src/main/java/marquez/core/mappers/CoreJobToApiJobMapper.java

```diff
@@ -6,12 +6,15 @@
 public class CoreJobToApiJobMapper extends Mapper<marquez.core.models.Job, Job> {
   @Override
   public Job map(marquez.core.models.Job value) {
-    // TODO: Update to add real values for input and output datasets
     return new Job(
         value.getName(),
         value.getCreatedAt(),
-        Collections.emptyList(),
-        Collections.emptyList(),
+        value.getInputDatasetUrns().isEmpty()
```

Comment:
is this ternary necessary? won't it accomplish the same thing if you just pass `value.getInputDatasetUrns()`? It will either be an empty, or a non-empty list

Reply:
Good point. Removed :)